ghec-audit-log-utils.js

```diff
@@ -15,7 +15,7 @@ function validateInput (program, config) {
   }
 
   // Validate correctness
-  const alphanumericRegex = /^[a-z0-9]+$/i
+  const tokenRegex = /^[a-z0-9_]+$/i
```

Comment:
I know `/i` includes this, but the official format is `[A-Za-z0-9_]` as highlighted in https://github.blog/changelog/2021-03-04-authentication-token-format-updates/

Should we keep it the same?